packages/three-mmd/src/loaders/MMDLoader.ts

```diff
@@ -62,8 +68,13 @@ import { GeometryBuilder } from './mmd-loader/geometry-builder'
  *  - more precise grant skinning support.
  *  - shadow support.
  */
+type BoneMesh = SkinnedMesh<any, any, any> & {
```

Comment:
I believe we should directly use SkinnedMesh and ignore the bones error.

Reply:
ok